docs/topic/secrets.md

```diff
@@ -0,0 +1,50 @@
+# Secrets and private keys
+
+(secrets:locations)=
+## Where are secrets stored
+
+Most secrets are stored in one of two locations:
+
+https://github.com/2i2c-org/infrastructure/blob/master/config/secrets.yaml
+: Secrets that are shared across all of our hub and cluster deployments.
+
+https://github.com/2i2c-org/pilot-hubs/tree/master/secrets
+: Secrets that are specific to each cluster / hub that we run (one JSON file for each cluster). For example, `Auth0` and proxy secrets.
```

Comment:
> For example, `Auth0` and proxy secrets.

This is incorrect, Auth0 and the secret key (not proxy secret) live in the above config file (these are general stuff, not cluster-specific stuff): https://github.com/2i2c-org/infrastructure/blob/master/config/secrets.yaml, you can see the keys without decrypting the file.

Reply:
I just tried to update this - does that look better?